core/app/models/spree/calculator.rb

```diff
@@ -1,7 +1,11 @@
 # frozen_string_literal: true
 
+require 'spree/preferences/persistable'
```

Comment:
I think this will be handled by the Rails autoloader, or is there a reason why this is needed explicitly?

Reply:
No, `spree/preferences/*` live in `lib`, and that isn't in the autoload path afaik. I did have to add it to make the tests pass :)